releases/v25_0_0.rst

```diff
@@ -1,17 +1,302 @@
+* `felis <https://github.com/lsst/felis>`_ contains DBMS-agnostic schema descriptions
+* `gbdes <https://github.com/lsst/gbdes>`_ is a fork of the gbernstein/gbdes package, modified for use by the LSST Science Pipelines as a third-party dependency.
+
+.. _release-v25-0-0-packages-removed:
+
+Packages Removed
+----------------
+* `Eigen <https://github.com/lsst/eigen>`_, replaced by the conda version.
+* `ap_pipe_testdata <https://github.com/lsst/ap_pipe_testdata>`_, obsolete gen2 test data.
+
+.. _release-v25-0-0-pending-deprecations:
+
+Pending Deprecations
+====================
+These Tasks or methods will be deprecated in the next major release.
+
+- :ref:`release-v25-0-0-deprecate-obs-base`
+- :ref:`release-v25-0-0-deprecate-pipe-tasks`
+- :ref:`release-v25-0-0-deprecate-ip-diffim`
+- :ref:`release-v25-0-0-deprecate-afw-table`
+- :ref:`release-v25-0-0-deprecate-daf-butler`
+- :ref:`release-v25-0-0-deprecate-configurations`
+
+.. _release-v25-0-0-deprecate-obs-base:
+
+Deprecate Method getInstrument
+------------------------------
+``obs.base.util.getInstrument`` will be replaced by ``lsst.pipe.base.Instrument.from_string``
+and will be removed after release of release v27.0.0.
+
+.. _release-v25-0-0-deprecate-pipe-tasks:
+
+Deprecate Interfaces ParquetTable and MultilevelParquetTable
+------------------------------------------------------------
+The Gen 2 I/O interfaces ``ParquetTable`` and ``MultilevelParquetTable`` in ``pipe_tasks``
+will be removed after the release of v26.0.0.
+
+.. _release-v25-0-0-deprecate-ip-diffim:
+
+Deprecate Factory Construction with makeIdFactory
+-------------------------------------------------
+ID factory construction with ``lsst.ip.diffim.detectAndMeasure.makeIdFactory`` should be replaced
+by using ``idGenerator`` config field and will be removed after the release of v27.0.0.
+
+.. _release-v25-0-0-deprecate-afw-table:
+
+Deprecate Method get_bool_array
+-------------------------------
+The ``get_bool_array`` method in ``lsst.afw.table.BaseColumnView``
+and its derived subclasses should not be used as
+``Catalog.__getitem__`` now provides better support accessing flag/bool columns and
+will be removed after the release of v27.0.0.
+
+.. _release-v25-0-0-deprecate-daf-butler:
+
+Pending Deprecations in lsst.daf.butler
+-----------------------------------------
+The methods ``Butler.put``, ``Butler.get`` and ``Butler.getDeferred``
+should be replaced by ``Butler.putDirect``, ``Butler.getDirect`` and ``Butler.getDeferred``
+``Butler.datasetExistsDirect`` will be replaced by ``Butler.stored``.
+``Butler.datasetExists`` will be replaced by ``Butler.exists``.
+The ``Butler.datastore`` property will be deprecated.
+The ``DimensionPacker`` class will be deprecated in v26.0.0 in favor of configurable dimension packers.
+Before mentioned will be removed after the release of v27.0.0.
+
+.. _release-v25-0-0-deprecate-skymap-packers:
+
+Deprecate SkyMapDimensionPacker
+--------------------------------------------
```

Comment:
Check that section heading underlines are the same length as the section (check all headings).